src/ardrive.ts

```diff
@@ -44,19 +48,15 @@ export class ArDriveAnonymous extends ArDriveType {
 	}
 }
 
-// TODO: ArDrive should accept App-Name and App-Version tags from constructor?
-const commTipMetaTags: GQLTagInterface[] = [
-	{ name: 'App-Name', value: 'ArDrive-CLI' },
-	{ name: 'App-Version', value: '2.0' },
-	{ name: 'Tip-Type', value: 'data upload' }
-];
-
 export class ArDrive extends ArDriveAnonymous {
 	constructor(
 		private readonly wallet: Wallet,
 		private readonly walletDao: WalletDAO,
 		protected readonly arFsDao: ArFSDAO,
-		private readonly communityOracle: CommunityOracle
+		private readonly communityOracle: CommunityOracle,
+		private readonly appName: string,
```

Comment:
We should also remember to feed these down into arfsDao. Currently all transactions are still being tagged with 

```
        {
          "name": "App-Name",
          "value": "ArDrive-Core"
        },
        {
          "name": "App-Version",
          "value": "1.0"
        },
```